aptos-move/block-executor/src/view.rs

```diff
@@ -2,7 +2,9 @@
 // SPDX-License-Identifier: Apache-2.0
```

Comment:
This file is getting really out of control and becoming like aptos_vm.rs as @georgemitenkov also noted.
@ziaptos you feel acceptable to land a refactor later, or should we break it down first (or here).

Reply:
let's do things in order:
- have it in this PR as is
- first we need to do some refactoring to fix the issues it has - new tests show failures here
- and then refactor across files

Reply:
sounds good.